### PR TITLE
feat(text-align-panel): 新增文字对齐配置组件

### PR DESCRIPTION
**Pull request by lijinke666**
### 👀 PR includes

✨ Feature

- [x] New feature

### 📝 Description

```tsx
  <TextAlignPanel
    title="文字对齐"
    defaultCollapsed={false}
    onChange={(options, theme) => {
      console.log('onChange:', options, theme);
    }}
    onReset={(options, prevOptions, theme) => {
      console.log('onReset:', options, prevOptions, theme);
    }}
  />
```

Kapture 2024-06-17 at 16 18 48

image

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-react-components/src/components/common/tooltip-wrapper/index.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { S2_PREFIX_CLS } from '@antv/s2';
```
Comment (lijinke666) [inline packages/s2-react-components/src/components/common/tooltip-wrapper/index.tsx:1]: image

File: packages/s2-react-components/src/components/config/text-align-panel/index.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+    const onOptionsChange =
+      (field: keyof TextAlignPanelOptions) => (e: RadioChangeEvent) => {
+        const newOptions: TextAlignPanelOptions = {
+          ...options,
+          [field]: e.target.value,
+        };
+
+        // 指标和列头对齐 (含 EXTRA_FIELD 虚拟列), 可单独调整指标
+        if (field === 'colCellTextAlign') {
```
Comment (lijinke666) [inline packages/s2-react-components/src/components/config/text-align-panel/index.tsx:46]: image

File: packages/s2-react-components/src/components/config/text-align-panel/utils.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  options: TextAlignPanelOptions,
+): S2Theme {
+  const {
+    colCellTextAlign: colCellAlignType,
+    rowCellTextAlign: rowCellAlignType = 'left',
+    dataCellTextAlign: dataCellAlignType = 'right',
+  } = options;
+
+  return {
+    // 角头取列头的对齐方式, 但底表默认行角头是靠左, 列角头是靠右, 所以没有指定默认值时默认按底表对齐方式展示.
```
Comment (lijinke666) [inline packages/s2-react-components/src/components/config/text-align-panel/utils.ts:18]: image

**Review by wjgogogo — APPROVED** [review]

**Pull request merged**